examples/rules.dl

```diff
@@ -0,0 +1,48 @@
+// Is this symbol an access path to some data?
+.decl isAccessPath(x: symbol)
+// Is this symbol a label.
+.decl isLabel(x: symbol)
+// A data flow edge
+.decl edge(src: symbol, tgt: symbol)
+ // A data flow path
+.decl path(src: symbol, tgt: symbol)
+// Definitely has this label.
+.decl hasLabel(v: symbol, label: symbol)
+// Definitely has this label.
+.decl claimHasLabel(v: symbol, label: symbol)
```

Comment:
(Because you mentioned that this is a good time to figure out terminology)

I think here a "label" is an element of a set, but to be consistent with IFC terminology, the labels should be the sets and we need another term for the elements of a set. Essentially, the IFC "flowsTo" relation is "is subset or equal to" on the sets (e.g., `{Alice}` is a label that says the data has only info owned by `Alice` can flowTo a location labeled `{Alice, Bob}`). 

Some other terms we could use for the elements of the set could be "tag" ([Flume](https://pdos.csail.mit.edu/papers/flume-sosp07.pdf) used the term tag for these -- Flume also used a label model where labels are drawn from the lattice over sets with subsetEq as the order) or "labelElement", or maybe "dataIdentifier".